MapboxDirectionsTests/IntructionsTests.swift

```diff
@@ -67,27 +68,24 @@ class SpokenInstructionsTests: XCTestCase {
         
         let spokenInstructions = step.instructionsSpokenAlongStep!
         
-        XCTAssertEqual(spokenInstructions[0].distanceAlongStep, 944.9)
-        XCTAssertEqual(spokenInstructions[1].distanceAlongStep, 258.0)
-        XCTAssertEqual(spokenInstructions[2].distanceAlongStep, 55.3)
-        
-        XCTAssertEqual(spokenInstructions[0].ssmlText, "<speak><amazon:effect name=\"drc\"><prosody rate=\"1.08\">Continue on Fillmore Street for a half mile</prosody></amazon:effect></speak>")
-        XCTAssertEqual(spokenInstructions[1].ssmlText, "<speak><amazon:effect name=\"drc\"><prosody rate=\"1.08\">In 900 feet, turn left onto <phoneme ph=\"ˈheɪt ˌstɹiːt\">Haight Street</phoneme></prosody></amazon:effect></speak>")
-        XCTAssertEqual(spokenInstructions[2].ssmlText, "<speak><amazon:effect name=\"drc\"><prosody rate=\"1.08\">Turn left onto <phoneme ph=\"ˈheɪt ˌstɹiːt\">Haight Street</phoneme></prosody></amazon:effect></speak>")
```

Comment:
Consider adding assertions like these to ensure that we can still handle multiple spoken instructions on a single step.

Reply:
Added back https://github.com/mapbox/MapboxDirections.swift/pull/243/commits/babece99b5f76f561d1686a5a932f9c9797c873b